src/routes/albumRoutes.js

```diff
@@ -1,5 +1,7 @@
 const express = require('express');
 const router = express.Router();
+const albumController = require('./albumController');
```

Comment:
Instead of requiring the full `albumController` we can just add the handler to the code on line 10-15 to follow the existing pattern in this file

Reply:
Done , commit 6bd3c85395f33ef870020c4635eeb7e68f3fc220